src/amo/components/AddonBadges/styles.scss

```diff
@@ -7,19 +7,13 @@
 
   @include respond-to(large) {
     display: flex;
-    flex-flow: row wrap;
+    flex-flow: column;
+    position: absolute;
```

Comment:
It seems weird to absolutely position something and have it be a flex container.

Reply:
yes, I think we can get rid of `display: flex`, if we use `position: absolute`